modules/connatixBidAdapter.js

```diff
@@ -308,6 +330,21 @@ export const spec = {
       params['us_privacy'] = encodeURIComponent(uspConsent);
     }
 
+    window.addEventListener('message', function handler(event) {
+      if (!event.data || !event.origin.includes('connatix')) {
```

Comment:
Since we're calling `stopImmediatePropagation` we ought to better check that the source is our connatix identity iframe before we do that. And some defensive programming anyways, since anybody can spoof an origin to contain connatix.

Recommended -> full url for iframe origin (spoofing protection) // additional payload data that strictly identifies our identity post message (protection in case some post message event from another connatix resource is intercepted and stopped by this code)

Reply:
Full url including query params?

Reply:
I replaced with the full URL (i think that's the right one, let me know if i'm mistaken), should an additional payload data be added as well? @mariusszabo